src/views/Entries/FilterEntriesForm/FrameworkFilter/index.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SearchInput from '#rsci/SearchInput';
+import DateFilter from '#rsci/DateFilter';
+import TimeFilter from '#rsci/TimeFilter';
+import RangeFilter from '#rsci/RangeFilter';
+import MultiSelectInput from '#rsci/MultiSelectInput';
+
+import _ts from '#ts';
+
+import GeoFilter from '../GeoFilter';
+
+import styles from './styles.scss';
+
+const emptyList = [];
+
+function FrameworkFilter(props) {
+    const {
+        title,
+        filterKey,
+        filter,
+        handleFilterChange,
+        value,
+        disabled,
+        regions,
+        geoOptions,
+    } = props;
+
+    if (!filter || !filter.type) {
+        return null;
+    }
+
+    const propsForFilter = {
+        key: filterKey,
+        className: styles.entriesFilter,
+        label: title,
+        onChange: values => handleFilterChange(filterKey, values),
+        value,
+        disabled,
+    };
+    switch (filter.type) {
+        case 'geo': {
+            return (
+                <GeoFilter
+                    {...propsForFilter}
+                    value={propsForFilter.value}
+                    disabled={propsForFilter.disabled}
+                    geoOptions={geoOptions}
+                    regions={regions}
+                />
+            );
+        }
+        case 'multiselect':
+            return (
+                <MultiSelectInput
+                    {...propsForFilter}
+                    value={propsForFilter.value || emptyList}
+                    options={filter.options || emptyList}
```

Comment:
Do we need this?

Reply:
By this I mean setting default values.
Also, why not use `??` syntax? I think it is already available in `develop`.